src/services/utilities.ts

```diff
@@ -800,6 +800,11 @@ namespace ts {
             return contextualType;
         }
 
+        const parent = node.parent;

```

Comment:
Is it all safe to put this new code into getContextualType? What happens if you put it there?